src/pages/components/pane.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Pane
+description: >
+  Panes give users a quick way to resize layouts vertically or horizontally by
+  interacting with the splitter. They can also be used to create expandable or
+  collapsible layouts.
+package: '@zendeskgarden/react-grid'
+components:
+  - grid/src/elements/pane/PaneProvider.tsx
+  - grid/src/elements/pane/Pane.tsx
+subcomponents:
+  - grid/src/elements/pane/components/Content.tsx
+  - grid/src/elements/pane/components/Splitter.tsx
+  - grid/src/elements/pane/components/SplitterButton.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To make a structured layout resizable, expandable or collapsible</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Vertical
+
+Panes can be used in a vertical orientation. The panes resize up to the values
+that you set.
+
+import PaneVertical from '../../examples/pane/PaneVertical.tsx';
+import PaneVerticalCode from '!!raw-loader!../../examples/pane/PaneVertical.tsx';
+
+<CodeExample code={PaneVerticalCode}>
+  <PaneVertical />
+</CodeExample>
+
+### Horizontal
+
+Panes can be used in a horizontal orientation as well.
+
+import PaneHorizontal from '../../examples/pane/PaneHorizontal.tsx';
+import PaneHorizontalCode from '!!raw-loader!../../examples/pane/PaneHorizontal.tsx';
+
+<CodeExample code={PaneHorizontalCode}>
+  <PaneHorizontal />
+</CodeExample>
+
+### Grid
+
+Panes can be resized in both vertical and horizontal directions for a complex
+grid flexible layout.
+
+import PaneGrid from '../../examples/pane/PaneGrid.tsx';
+import PaneGridCode from '!!raw-loader!../../examples/pane/PaneGrid.tsx';
+
+<CodeExample code={PaneGridCode}>
+  <PaneGrid />
+</CodeExample>
+
+## Customization
+
+### Vertical collapse
+
+Panes can be collapsed or expanded by using the splitter together with an
+[Icon button](/components/icon-button). Vertical orientation uses a small circular
+icon button positioned on top of the splitter by default in both collapsed and
+expanded states. [Tooltips](/components/tooltip) should be used to describe the action.
```

Comment:
Ah I see what you mean. @lucijanblagonic / @diana-oum  any thoughts here?

Reply:
Thinking back how this was initially documented, the expand collapse was supposed to be a pattern, so I thought referencing components used to build it made sense. But if we don't want users thinking about the actual components being used (as they come packaged out of the box) to build this I am OK with the change. If @diana-oum can give it a look 👀, that would be great. 

The only thing I am not sure is, should we just say "action" instead of the button / small circular button. As with this "small circular button" we are being oddly specific and it might not be necessary if it's not going to be referenced (linked) as a small icon button. 🤔

Reply:
Just a suggestion with the language - using the component name `SplitterButton` could be a good replacement (somewhat self explanatory) and would go a long way in tying the action/button within the context of `Pane` (without going deep into the implementation). As a developer, it makes it easier to mentally follow this down to the API section and identify the component in the example code.

I do like that the location of the action button / `SplitterButton` is described, I think that will be very helpful to anyone new to the component.

Reply:
➕ 1 to Mike's suggestion to using its actual name. I think thats the most explicit and clear option for our readers to understand how to implement these things using the things we provide them with.

> Panes can be collapsed or expanded by using the SplitterButton[hyperlink to splitter button API section]. Vertical orientation uses a small circular icon button positioned on top of the splitter by default in both collapsed and expanded states. Include a label to describe the action.

The last sentence I am not sure though. What guidance are we giving for the label? Are we saying people should write a descriptive label for the action when they're using the component or to make sure to have a label in the implementation as it is an optional part to include? 

@lucijanblagonic Will there be a tooltip in the demo example? I don't see it in the Figma file. We should make sure we're providing good content guidance for the tooltip. Let's work on that if so.

Reply:
> The last sentence I am not sure though. What guidance are we giving for the label? Are we saying people should write a descriptive label for the action when they're using the component or to make sure to have a label in the implementation as it is an optional part to include?

@diana-oum `label` is a required prop for `SplitterButton` that the consumer needs to supply. The API section describes this clearly so I'd say we don't repeat here, unless you wanted to provide content guidance for it.

Here is my suggestion, (less visual description to decouple our documentation from visual design).

> Panes can be collapsed or expanded by using the SplitterButton[hyperlink to splitter button API section].

If we want to specify the default positions of the icon button w.r.t. the splitter, then the **Icon button position** section feels like the better place for it. See [comment](https://github.com/zendeskgarden/website/pull/416/files#r938270527).

I'd defer to @diana-oum for the final call.

Reply:
@smritimadan Looks amazing, Smriti! Great suggestions, thank you